core/src/main/java/com/scalar/db/common/ActiveTransactionManagedTwoPhaseCommitTransactionManager.java

```diff
@@ -30,14 +31,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public abstract class ActiveTransactionManagedTwoPhaseCommitTransactionManager
-    extends TransactionDecorationTwoPhaseCommitTransactionManager
-    implements TwoPhaseCommitTransactionExpirationHandlerSettable {
+public class ActiveTransactionManagedTwoPhaseCommitTransactionManager
```

Comment:
Ditto. Refactored this from using inheritance to using composition (decorator pattern).